api-spec/STAC-standalone.yaml

```diff
@@ -21,7 +21,20 @@ servers:
   - url: 'http://www.cool-sat.com'
     description: Production server
 paths:
-  /search/stac:
+  /stac:

```

Comment:
The geometry in line 442 could also refer to the GEOJSON geometry, i.e.
```
        geometry:
          $ref: "http://json.schemastore.org/geojson.json#/definitions/geometry"
```
But that's a nice to have and by no way blocking. May also just be solved with #131.